src/test/scala/chiselTests/RecordSpec.scala

```diff
@@ -146,6 +146,70 @@ class RecordSpec extends ChiselFlatSpec with RecordSpecUtils with Utils {
     e.getMessage should include("contains aliased fields named (bar,foo)")
   }
 
+  they should "be OpaqueType for single element maps" in {
```

Comment:
There are lots of tests in here, can you split them up between different named blocks describing what each test is checking?